Cargo.toml

```diff
@@ -174,7 +174,8 @@ solang-parser = "=0.3.3"
 ## revm
 revm = { version = "14.0.3", default-features = false }
 revm-primitives = { version = "10.0.0", default-features = false }
-revm-inspectors = { version = "0.8.0", features = ["serde"] }
+# Update this after the PR https://github.com/paradigmxyz/revm-inspectors/pull/213 is merged
```

Comment:
this one was merged

Reply:
it hasn't been included in [the lastest release](https://github.com/paradigmxyz/revm-inspectors/tags) of `revm-inspectors`, should we wait for the next release first?

Reply:
hi @grandizzy I've merged the master branch, now it should have the latest `revm-inspectors` version